lib/aws/active_job/sqs/configuration.rb

```diff
@@ -139,16 +196,45 @@ def set_attributes(options)
           end
         end
 
+        # resolve ENV for global and queue specific options
+        def env_options
+          resolved = { queues: {} }
+          GLOBAL_ENV_CONFIGS.each do |cfg|
+            env_name = "AWS_ACTIVE_JOB_SQS_#{cfg.to_s.upcase}"
+            resolved[cfg] = parse_env_value(env_name) if ENV.key? env_name
+          end
+
+          # check for queue specific values
+          queue_key_regex =
+            /AWS_ACTIVE_JOB_SQS_([a-zA-Z0-9_-]+)_(#{QUEUE_ENV_CONFIGS.map(&:upcase).join('|')})/
+          ENV.each_key do |key|
+            next unless (match = queue_key_regex.match(key))
+
+            queue_name = match[1].downcase.to_sym
+            resolved[:queues][queue_name] ||= {}
+            resolved[:queues][queue_name][match[2].downcase.to_sym] =
+              parse_env_value(key)
+          end
+          resolved
+        end
+
+        def parse_env_value(key)
+          val = ENV.fetch(key, nil)
+          Integer(val)
+        rescue ArgumentError, TypeError
+          %w[true false].include?(val) ? val == 'true' : val
+        end
+
         def file_options(options = {})
-          file_path = config_file_path(options)
+          file_path = options[:config_file] || default_config_file
           if file_path
             load_from_file(file_path)
           else
-            {}
+            options
           end
         end
 
-        def config_file
+        def default_config_file
           file = ::Rails.root.join("config/aws_sqs_active_job/#{::Rails.env}.yml")
```

Comment:
Env name and config file no longer match. Can we normalize all usages to aws_active_job_sqs before the major version?